misc/docs/source/tutorial/code_snippets/stationxml_custom_tags.rst

```diff
@@ -0,0 +1,167 @@
+    print(inv[0].extra)
+
+.. code-block:: python
+
+    AttribDict({u'my_tag': AttribDict({'attrib': {'{http://some-page.de/xmlns/1.0}my_attrib2': '567',
+	                                              '{http://some-page.de/xmlns/1.0}my_attrib1': '123.4'},
+	                                              'namespace': 'http://some-page.de/xmlns/1.0', 
+	                                              'value': 'True'}), 
+	            u'my_tag_4': AttribDict({'namespace': 'http://test.org/xmlns/0.1',
+	                                     'value': '2013-01-02T13:12:14.600000Z'}), 
+	            u'my_attribute': AttribDict({'namespace': 'http://test.org/xmlns/0.1',
+	                                         'value': 'my_attribute_value'}),
+	            u'my_tag_2': AttribDict({'namespace': 'http://some-page.de/xmlns/1.0', 
+	                                     'value': 'True'}), 
+	            u'my_tag_3': AttribDict({'namespace': 'http://some-page.de/xmlns/1.0',
+	                                     'value': '1'})})
+
+Custom tags can be nested:
+
+.. code-block:: python
+
+	from obspy import Inventory
+	from obspy.core.inventory import Network
+	from obspy.core.util import AttribDict
+	
+	ns = 'http://some-page.de/xmlns/1.0'
+	
+	my_tag = AttribDict()
+	my_tag.namespace = ns
+	my_tag.value = AttribDict()
+	
+	my_tag.value.my_nested_tag1 = AttribDict()
+	my_tag.value.my_nested_tag1.namespace = ns
+	my_tag.value.my_nested_tag1.value = 1.23E+10
+	
+	my_tag.value.my_nested_tag2 = AttribDict()
+	my_tag.value.my_nested_tag2.namespace = ns
+	my_tag.value.my_nested_tag2.value = True
+	
+	inv = Inventory([Network('XX')], 'XX')
+	inv[0].extra = AttribDict()
+	inv[0].extra.my_tag = my_tag
+	inv.write('my_inventory.xml', format='STATIONXML',
+	          nsmap={'somepage_ns': 'http://some-page.de/xmlns/1.0'})
+
+This will produce an xml output similar to the following:
+
+.. code-block:: xml
+
+	<?xml version='1.0' encoding='UTF-8'?>
+	<FDSNStationXML xmlns:somepage_ns="http://some-page.de/xmlns/1.0" xmlns="http://www.fdsn.org/xml/station/1" schemaVersion="1.0">
+	  <Source>XX</Source>
+	  <Module>ObsPy 1.0.2</Module>
+	  <ModuleURI>https://www.obspy.org</ModuleURI>
+	  <Created>2016-10-17T18:45:14.302265+00:00</Created>
+	  <Network code="XX">
+	    <somepage_ns:my_tag>
+	      <somepage_ns:my_nested_tag1>12300000000.0</somepage_ns:my_nested_tag1>
+	      <somepage_ns:my_nested_tag2>True</somepage_ns:my_nested_tag2>
+	    </somepage_ns:my_tag>
+	  </Network>
+	</FDSNStationXML>
+
+The output xml can be read again using
+:meth:`read_inventory() <obspy.core.inventory.read_inventory>` and the nested tags can be
+retrieved in the following way:
+
+.. code-block:: python
+
+    from obspy import read_inventory
+
+    inv = read_inventory('my_inventory.xml')
+    print(inv[0].extra.my_tag.value.my_nested_tag1.value)
+    print(inv[0].extra.my_tag.value.my_nested_tag2.value)
+
+.. code-block:: python
+
+    12300000000.0
+    true
```

Comment:
Maybe a stupid question: Why it this lower case true and not capitalized like in the XML?